client/src/components/container/admin/controlPage/component/ViewEvents/ViewEvents.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import { Link } from "react-router-dom";
+import { Table, message, Spin } from 'antd';
+import axios from 'axios';
+
+import './ViewEvents.css'
+
+class ViewEvents extends Component {
+  state = {
+    error: null,
+    isLoaded: false,
+    events: [],
+    errorMessage:null,
+  }
+  componentDidMount() {
+    this.setState({ isLoaded: true })
+    axios.get('/api/v1/event').
+      then(({ data }) => {
+        this.setState({
+          isLoaded: false,
+          events: data,
+        });
+      }).catch(() => {
+        this.setState({ errorMessage: "Internal server error !!" })
+      })
+  }
+
+  handleAction = currentEvent => {
+    axios.patch("/event", { id : currentEvent })
+      .then(({data}) => {
+        console.log("done");
+        message.error(data.msg, 10);
+      })
+      .catch(() => {
+        console.error("error");
+      });
+  };
+
+  handelEventButtons = (record) => (
+    <div>
+      <Link to={`/dashboard/${record.event_code}/users`}>Show </Link>
+      <Link type="primary" onClick={() => this.handleAction(record.id)}>
+        Delete
+        </Link>
```

Comment:
I think here , by mistake you write `Link` instead of `Button` 
```suggestion
      <Button type="primary" onClick={() => this.handleAction(record.id)}>
        Delete
        </Button>
```